contrib/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ if (USE_INTERNAL_SSL_LIBRARY)
     add_library(OpenSSL::SSL ALIAS ${OPENSSL_SSL_LIBRARY})
 endif ()
 
-if (ENABLE_LDAP AND USE_INTERNAL_LDAP_LIBRARY)
```

Comment:
There is table `system.build_options`, I think it would be useful to add `USE_LDAP` in there.

Reply:
Done. Thanks.